docs/source/telegram.audio.rst

```diff
@@ -1,7 +1,6 @@
 telegram.audio module
 =====================
 
-.. automodule:: telegram.audio
+.. automodule:: telegram.files.audio
     :members:
-    :undoc-members:
```

Comment:
Would it not be safer to just leave the :undoc-members: instead of removing it in __some__ cases?